mod/account/set/right/password.rb

```diff
@@ -38,9 +38,16 @@ def ok_to_read
     render_raw
   end
 
+  def input_type
+    :password
+  end
+
+  def password_input
+    haml :password_input
+  end
+
   view :input do
-    card.content = ""
-    password_field :content, class: "d0-card-content", autocomplete: autocomplete?
+    password_input
   end
```

Comment:
*Either* you want to override the input view, *or* you want to use the `input_type` pattern. But not both. (The second option is probably preferable)

The thing that calls `input_type` is the input view. If you're overriding the input_view, then there is no need for `input_type`.

Reply:
I am not sure I understand. The input view automatically calls input_type? I don't see input_type called anywhere. Is this a naming convention? name_of_type gets called by view :name_of?